docs/tools/vdb_table/data/redis.json

```diff
@@ -149,15 +149,15 @@
     "comment": ""
   },
   "github_stars": {
-    "value": 34,
-    "source_url": "https://github.com/RedisAI/VectorSimilarity",
+    "value": 5200,
+    "source_url": "https://github.com/RediSearch/RediSearch",
```

Comment:
The initial reason for chosing the VectorSimilarity repo was that we couldn't find & confirm that RediSearch actually contains the vector similarity search functionality - could you point me to the right file @adrianoamaral ?

Reply:
https://github.com/RediSearch/RediSearch/tree/master/deps

Reply:
I see could you put that into the comment - for example "Vector Search included via https://github.com/RediSearch/RediSearch/tree/master/deps"

Reply:
Sorry @svonava.  Vector search is included in RediSearch as a git submodule as you can check here https://github.com/RediSearch/RediSearch/tree/master/deps .

Reply:
I understand that - what I'm saying is, let's mention that in the comment field on line 154 (that way, the readers who see RediSearch as the star source understand this context).

Reply:
Comments added @svonava